components/item/AwesomeSwiper.stories.js

```diff
@@ -0,0 +1,51 @@
+import { storiesOf } from '@storybook/vue';
+import VueI18n from 'vue-i18n';
+import AwesomeSwiper from './AwesomeSwiper';
+
+const i18n = new VueI18n();
+
+const media = [
+  { about: 'https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w400&type=IMAGE&uri=https%3A%2F%2Feuropeana1914-1918.s3.amazonaws.com%2Fattachments%2F119112%2F10265.119112.original.jpg' },
+  { about: 'https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w400&type=IMAGE&uri=https%3A%2F%2Feuropeana1914-1918.s3.amazonaws.com%2Fattachments%2F119200%2F10265.119200.original.jpg' },
+  { about: 'https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w400&type=IMAGE&uri=https%3A%2F%2Feuropeana1914-1918.s3.amazonaws.com%2Fattachments%2F119203%2F10265.119203.original.jpg' },
+  { about: 'https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w400&type=IMAGE&uri=https%3A%2F%2Feuropeana1914-1918.s3.amazonaws.com%2Fattachments%2F119639%2F10265.119639.original.jpg' },
+  { about: "https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w400&type=IMAGE&uri=https%3A%2F%2Feuropeana1914-1918.s3.amazonaws.com%2Fattachments%2F119640%2F10265.119640.original.jpg" }
+];
+
+storiesOf('Item page/Awesome Swiper', module)
+  .add('Centered with multiple slides visible', () => ({
+    i18n,
+    components: { AwesomeSwiper },
+    data() {
+      return {
+        media,
+        id: '/2020601/https___1914_1918_europeana_eu_contributions_10265'
+      };
+    },
+    template: `
+      <div class="mt-3">
+        <AwesomeSwiper
+          :europeana-identifier="id"
+          :media="media"
+        />
+      </div>
+    `
+  }))
+  .add('Full height images', () => ({
```

Comment:
let's get rid of this as it's just the same as the other example at the moment.

Reply:
ok